libwebp version 1.3.1 breaks build -  fix provided

### Did you read the guide?

Yes

### What is the base version number of Motion being used?

4.5.x (really it's git HEAD)

### What was the install method?

Built from source code

### What is base architecture?

x86

### What is the distro being used?

Other

### Disto version number

Arch linux Fully updated

### Camera type(s) being used?

Network camera with RTSP

### Describe the issue/problem and steps to reproduce

Build started failing with missing webp function. Readon is configure adds libwebpmux but it  now also needs libwebp for the missing function. This broke with the update to ibwebp 1.3.1.  Version 1.3.0 was fine.

  `/usr/bin/ld: picture.o: undefined reference to symbol 'WebPMemoryWriterInit'`

Fix that worked is to change configure.ac from:

  `TEMP_LIBS="$TEMP_LIBS "`pkg-config --libs libwebpmux ``

 to

  `TEMP_LIBS="$TEMP_LIBS "`pkg-config --libs libwebpmux libwebp``

### Motion log output at log_level 8

```shell
N/A
```

--- a/configure.ac
+++ b/configure.ac
@@ -191,8 +191,8 @@ AS_IF([test "${WEBP}" = "yes" ], [
     AS_IF([pkg-config libwebpmux ], [
         AC_MSG_RESULT(yes)
         AC_DEFINE([HAVE_WEBP], [1], [Define to 1 if WEBP is around])
-        TEMP_CFLAGS="$TEMP_CFLAGS "`pkg-config --cflags libwebpmux`
-        TEMP_LIBS="$TEMP_LIBS "`pkg-config --libs libwebpmux`
+        TEMP_CFLAGS="$TEMP_CFLAGS "`pkg-config --cflags libwebpmux libwebp`
+        TEMP_LIBS="$TEMP_LIBS "`pkg-config --libs libwebpmux libwebp`
       ],[
         AC_MSG_RESULT(no)
       ]

